{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "3f76e61e_bda59bc4",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 10
      },
      "lineNbr": 0,
      "author": {
        "id": 1000049
      },
      "writtenOn": "2023-01-19T18:25:06Z",
      "side": 1,
      "message": "Not sure if some of the failures were due to a rebase being necessary. Rebasing and re running CI.",
      "revId": "4184097c4eeaef3845facee4cff44a6e56f78c44",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    }
  ]
}